src/components/Table/index.tsx

```diff
@@ -211,7 +211,7 @@ class Table extends React.Component<TableProps, TableState> {
     }
 
     private renderHead(row: CellData[]) {
-        const cells = row.map((c, index) => <th key={index}>{c}</th>);
+        const cells = row.map((c, index) => c !== '' ?  <th key={index}>{c}</th> : <th key={index}>&nbsp;</th>);
```

Comment:
in case **c** is `null` should we return `{index}` or `&nbsp;`?

the point is do we have to check `c != ''` or we can `!c`?

Reply:
when we have close icon on table data, it is overlapping on table header, so now we have just empty th in this case

Reply:
@pkhlopiachyi Use !c.length